entries/w/wellsfargo.com.json

```diff
@@ -9,7 +9,14 @@
     "tfa": [
```

Comment:
Not sure if they provide 2FA.

>To protect your personal information and guard against unauthorized access to your accounts, we may require a code when you:
>* Access certain Wells Fargo accounts or services.
> * Send money to another person that you haven’t transferred money to before, or receive money from a non-Wells Fargo account that you haven’t used before.

Reply:
According to https://www.wellsfargo.com/privacy-security/fraud/articles/two-factor-authentication/, they do:
> Customers can also add an additional layer of security by activating Wells Fargo's 2FA feature, 2-Step Verification at Sign-On.